packages/index-converter/src/util/xml-processing-util.js

```diff
@@ -251,6 +251,18 @@ module.exports = {
         }
         return jsonObject2;
     },
+
+    copyAnalyzersFolder: (basePath, transformationMap) =>{
+        for (let entry of transformationMap.entries()) {
+           let analyzersFolderPath = path.join(basePath,entry[0],constants.INDEX_ANALYZERS);
+           fs.exists(analyzersFolderPath,(exists) =>{
```

Comment:
You should use `fs.existsSync` here, since you want the result before proceeding to the next steps (i.e. synchronously).

Reply:
I tried with `fs.existsSync` , but it is not working as expected with Sync , and it is working fine in async . Please let me know if you have any other opinion on this.